src/app/public/modules/split-view/split-view-media-query.service.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  SkyMediaBreakpoints,
+  SkyMediaQueryListener
+} from '@skyux/core';
+
+import {
```

Comment:
Can we possibly combine the two `rxjs` imports into one?

Reply:
Done.